syntax = "proto3";

package util;

option java_package = "common.model.protoUtils";

message DecimalValueProto {
  uint32 scale = 1;
  uint32 precision = 2;
  bytes value = 3;
}